### PR TITLE
Punctuation fix: Add missing period

### PR DESCRIPTION
**Pull request by Overlandcoder**
### Description

Added missing period
### Motivation

Improves punctuation

### CODE REVIEW

File: files/en-us/learn/forms/form_validation/index.md [hunk 1]
```diff
@@ -442,7 +442,7 @@ The Constraint validation API makes the following properties available on the ab
 The Constraint Validation API also makes the following methods available on the above elements and the [`form`](/en-US/docs/Web/HTML/Element/form) element.
 
 - `checkValidity()`: Returns `true` if the element's value has no validity problems; `false` otherwise. If the element is invalid, this method also fires an [`invalid` event](/en-US/docs/Web/API/HTMLInputElement/invalid_event) on the element.
-- `reportValidity()`: Reports invalid field(s) using events. Useful in combination with `preventDefault()` in an `onSubmit` event handler
+- `reportValidity()`: Reports invalid field(s) using events. Useful in combination with `preventDefault()` in an `onSubmit` event handler.
```
Comment (dipikabh) [inline files/en-us/learn/forms/form_validation/index.md:445]:
```suggestion
- `reportValidity()`: Reports invalid field(s) using events. This method is useful in combination with `preventDefault()` in an `onSubmit` event handler.
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks for this PR, @Overlandcoder. While we're at it, let's make the sentence complete to match other descriptions.

**Review by dipikabh — APPROVED** [review]
Thank you! 👍

**Comment by schalkneethling** [comment]
Welcome to the community, @Overlandcoder 🎉 If you are interested in more ways to contribute to MDN Web Docs, please have a look at our [contributor task board](https://github.com/orgs/mdn/projects/25). If you have any questions or need help finding something to work on, feel free to ask for help in our [community discussions](https://github.com/orgs/mdn/discussions). We look forward to more of your contributions. 🙌

**Pull request merged**